src/webgl/light.js

```diff
@@ -113,27 +113,40 @@ p5.prototype.ambientLight = function(v1, v2, v3, a) {
  * @example
  * <div>
  * <code>
+ * let setSpecularColor = true;
+ *
  * function setup() {
  *   createCanvas(100, 100, WEBGL);
  *   noStroke();
  * }
  *
  * function draw() {
  *   background(0);
- *   shininess(20);
- *   ambientLight(50);
- *   specularColor(255, 0, 0);
- *   pointLight(255, 0, 0, 0, -50, 50);
- *   specularColor(0, 255, 0);
- *   pointLight(0, 255, 0, 0, 50, 50);
- *   specularMaterial(255);
- *   sphere(40);
+ *
+ *   ambientLight(60);
+ *
+ *   // add point light to showcase specular color
+ *   let locX = mouseX - width / 2;
+ *   let locY = mouseY - height / 2;
+ *   if (setSpecularColor) {
```

Comment:
maybe conditional should go under the pointlight call so that the comment above reads like the intro to a relevant chunk?